packages/devtools_app/lib/src/info/info_controller.dart

```diff
@@ -18,6 +18,19 @@ class InfoController extends DisposableController
     _listenForFlutterVersionChanges();
   }
 
+  /// Return the FlutterVersion for the current connected app, or `null`
+  /// if the `flutterVersion` service extension does not exist.
+  static Future<FlutterVersion> getFlutterVersion() async {
```

Comment:
Avoid get in method name

Reply:
This is delegating to the existing `serviceManager.getFlutterVersion()` call; I'd suggest if we choose to update we do that in a separate PR.